### ISSUE TITLE
Support results larger than f64

### ISSUE THREAD

**Issue by izissise** [NerdyPepper/eva#47]
It would be nice if the calcul is larger than the size of f64 it display the exact value

**Comment by pickfire** [comment]
Not exactly, https://github.com/NerdyPepper/eva/issues/34 would be more useful to display the exact value that you see in calculator.

**Comment by JensGrabner** [comment]
In Planmaker from [Softmaker](https://www.softmaker.de/) have a correct result.

**Comment by pickfire** [comment]
I tried doing this with `num`'s `Ratio`, it doesn't support `f64` methods like `sin` so I am not sure what to do here.
